tmt/steps/execute/__init__.py

```diff
@@ -58,6 +59,9 @@
 # Scripts source directory
 SCRIPTS_SRC_DIR = tmt.utils.resource_files('steps/execute/scripts')
 
+# Scripts destination directory, hardcoded in the tmt.sh script
+SCRIPTS_DEST_DIR = "/var/tmp/tmt/bin"
```

Comment:
It's a path, it shall be a `Path` instance.

Reply:
And I'd add a way to override this path right from the beginning, check out `effective_pidfile_root()`, but that would conflict with easy hard-coding the path into `tmt.sh`... Still, hard-coded paths turned out to not be generally usable, sooner or later, might be a good investment to get this one configurable while we're not [ushed by a looming regression...

Reply:
Makes sense. Let's make it configurable.

Reply:
should be done.